src/Symfony/Component/DependencyInjection/Tests/Dumper/PhpDumperTest.php

```diff
@@ -1976,6 +1976,115 @@ public function testCallableAdapterConsumer()
         $this->assertInstanceOf(SingleMethodInterface::class, $container->get('bar')->foo);
         $this->assertInstanceOf(Foo::class, $container->get('bar')->foo->theMethod());
     }
+
+    /**
+     * @dataProvider getStripCommentsCodes
+     */
+    public function testStripComments(string $source, string $expected)
+    {
+        $container = new ContainerBuilder();
+        $container->compile();
+        $dumper = new PhpDumper($container);
+
+        $reflection = new \ReflectionClass($dumper);
+        $method = $reflection->getMethod('stripComments');
+        $method->setAccessible(true);
```

Comment:
`setAccessible` is useless. It is a no-op in PHP 8.1+

Reply:
I didn't know, great !